--- origsrc/gtranslator-1.9.5/configure.ac	2009-04-10 20:03:51.000000000 -0500
+++ src/gtranslator-1.9.5/configure.ac	2009-04-13 01:19:45.932625000 -0500
@@ -112,7 +112,7 @@ AC_PATH_PROG([GLIB_MKENUMS], [glib-mkenu
 dnl -----------------------------------
 dnl Plugins libraries compilation flags
 dnl -----------------------------------
-PLUGIN_LIBTOOL_FLAGS="-module -avoid-version"
+PLUGIN_LIBTOOL_FLAGS="-module -avoid-version -no-undefined \$(top_builddir)/src/libgtranslator.la"
 AC_SUBST(PLUGIN_LIBTOOL_FLAGS)
 
 dnl -------------------------------------------------------------------
--- origsrc/gtranslator-1.9.5/data/gtranslator.pc.in	2009-03-09 00:23:04.000000000 -0500
+++ src/gtranslator-1.9.5/data/gtranslator.pc.in	2009-04-13 11:34:05.760750000 -0500
@@ -8,3 +8,4 @@ Description: gtranslator
 Requires: libglade-2.0 gtksourceview-2.0
 Version: @VERSION@
 Cflags: -I${includedir}/gtranslator-@GTR_API_VERSION@
+Libs: -L${libdir} -lgtranslator
--- origsrc/gtranslator-1.9.5/src/Makefile.am	2009-03-24 00:26:12.000000000 -0500
+++ src/gtranslator-1.9.5/src/Makefile.am	2009-04-13 01:19:21.260750000 -0500
@@ -7,7 +7,7 @@ SUBDIRS = toolbareditor \
 	  plugin-system \
 	  translation-memory
 
-noinst_LTLIBRARIES = libgtranslator.la
+lib_LTLIBRARIES = libgtranslator.la
 
 INCLUDES = \
 	-I$(top_srcdir)						\
@@ -112,6 +112,7 @@ libgtranslator_la_SOURCES = \
 
 libgtranslator_la_LIBADD = \
 	$(GTRANSLATOR_LIBS) \
+	$(GTKSPELL_LIBS) \
 	dialogs/libdialogs.la \
 	toolbareditor/libtoolbareditor.la \
 	plugin-system/libpluginsystem.la \
