--- origsrc/gtranslator-1.9.6/configure.ac	2009-08-24 10:40:43.000000000 -0500
+++ src/gtranslator-1.9.6/configure.ac	2009-09-09 02:47:39.168302100 -0500
@@ -116,15 +116,30 @@ AC_MSG_CHECKING([for Win32 platform])
 case "$host" in
   *-*-mingw*|*-*-cygwin*)
     platform_win32=yes
-    AC_CHECK_TOOL(WINDRES, windres)
+    GTR_IMPLIB=-lgtranslator-$GTR_API_VERSION
     ;;
   *)
     platform_win32=no
+    GTR_IMPLIB=
     ;;
 esac
 AC_MSG_RESULT([$platform_win32])
+AC_SUBST(GTR_IMPLIB)
 AM_CONDITIONAL(PLATFORM_WIN32, test "$platform_win32" = "yes")
 
+AC_MSG_CHECKING([for native Win32])
+case "$host" in
+  *-*-mingw*)
+    os_win32=yes
+    AC_CHECK_TOOL(WINDRES, windres)
+    ;;
+  *)
+    os_win32=no
+    ;;
+esac
+AC_MSG_RESULT([$os_win32])
+AM_CONDITIONAL(OS_WIN32, test "$os_win32" = "yes")
+
 
 dnl -----------------------------------
 dnl Plugins libraries compilation flags
@@ -132,7 +147,7 @@ dnl -----------------------------------
 # on win32 plugins must be able to resolve all symbols at link time
 # so we create a lib
 if test "$platform_win32" = yes; then
-	PLUGIN_LIBTOOL_FLAGS="-module -avoid-version -no-undefined -mwindows -Wl,\$(top_builddir)/src/libgtranslator-\$(GTR_API_VERSION).a"
+	PLUGIN_LIBTOOL_FLAGS="-module -avoid-version -no-undefined -Wl,\$(top_builddir)/src/libgtranslator-\$(GTR_API_VERSION).a"
 else
 	PLUGIN_LIBTOOL_FLAGS="-module -avoid-version"
 fi
--- origsrc/gtranslator-1.9.6/data/gtranslator.pc.in	2009-08-19 15:17:54.000000000 -0500
+++ src/gtranslator-1.9.6/data/gtranslator.pc.in	2009-09-09 02:45:52.092916000 -0500
@@ -8,3 +8,4 @@ Description: gtranslator
 Requires: libglade-2.0 gtksourceview-2.0
 Version: @VERSION@
 Cflags: -I${includedir}/gtranslator-@GTR_API_VERSION@
+Libs: -L${libdir} @GTR_IMPLIB@
--- origsrc/gtranslator-1.9.6/src/Makefile.am	2009-08-19 15:17:54.000000000 -0500
+++ src/gtranslator-1.9.6/src/Makefile.am	2009-09-09 02:49:30.428752200 -0500
@@ -43,7 +43,7 @@ gtranslator_SOURCES = main.c
 gtranslator_LDADD = libgtranslator.la $(GTRANSLATOR_LIBS) $(GTKSPELL_LIBS) $(INTLLIBS)
 
 if PLATFORM_WIN32
-gtranslator_LDFLAGS = -Wl,--export-all-symbols -mwindows -Wl,--out-implib,libgedit-$(GTR_API_VERSION).a
+gtranslator_LDFLAGS = -Wl,--export-all-symbols -Wl,--out-implib,libgtranslator-$(GTR_API_VERSION).a
 else
 gtranslator_LDFLAGS = -export-dynamic -no-undefined -export-symbols-regex "^[[^_]].*"
 endif
@@ -57,7 +57,7 @@ BUILT_SOURCES =                         
 	gtranslator-enum-types.h \
 	gtranslator-enum-types.c
 
-if PLATFORM_WIN32
+if OS_WIN32
 gtranslator-res.o: gtranslator.rc
 	$(WINDRES) -i gtranslator.rc --input-format=rc -o gtranslator-res.o -O coff
 
@@ -152,6 +152,12 @@ gtranslator-marshal.h: gtranslator-marsh
 gtranslator-marshal.c: gtranslator-marshal.list $(GLIB_GENMARSHAL)
 	$(AM_V_GEN) $(GLIB_GENMARSHAL) $< --body --header --prefix=gtranslator_marshal > $@
 
+if PLATFORM_WIN32
+install-exec-hook:
+	$(mkinstalldirs) $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) libgtranslator-$(GTR_API_VERSION).a $(DESTDIR)$(libdir)
+endif
+
 EXTRA_DIST =	\
 	gtranslator-marshal.list	\
 	gtranslator-enum-types.h.template	\
