--- origsrc/gtranslator-2.90.4/src/Makefile.am	2011-05-25 15:19:58.120856400 -0500
+++ src/gtranslator-2.90.4/src/Makefile.am	2011-05-25 15:33:48.438347900 -0500
@@ -173,11 +173,11 @@ if HAVE_INTROSPECTION
 -include $(INTROSPECTION_MAKEFILE)
 INTROSPECTION_GIRS = Gtranslator-3.0.gir
 
-Gtranslator-3.0.gir: gtranslator
+Gtranslator-3.0.gir: gtranslator$(EXEEXT)
 INTROSPECTION_SCANNER_ARGS = -I$(top_srcdir) --warn-all --identifier-prefix=Gtr
 Gtranslator_3_0_gir_NAMESPACE = Gtranslator
 Gtranslator_3_0_gir_VERSION = 3.0
-Gtranslator_3_0_gir_PROGRAM = $(builddir)/gtranslator
+Gtranslator_3_0_gir_PROGRAM = $(builddir)/gtranslator$(EXEEXT)
 Gtranslator_3_0_gir_FILES = $(INST_H_FILES) $(libgtranslator_c_files)
 Gtranslator_3_0_gir_INCLUDES = Gtk-3.0 GtkSource-3.0
 
